tests/Uri/UriRetrieverTest.php

```diff
@@ -339,6 +339,28 @@ public function testJsonSchemaOrgMediaTypeHack()
         $this->assertTrue($retriever->confirmMediaType($mock, 'http://json-schema.org/'));
     }
 
+    /**
+     * @expectedException \JsonSchema\Exception\InvalidSchemaMediaTypeException
+     */
+    public function testJsonSchemaOrgMediaTypeBlacklistUnknown()
```

Comment:
Naming - not a blacklist, and nothing to do with json-schema.org.

Reply:
done